pip/commands/install.py

```diff
@@ -374,6 +375,24 @@ def run(self, options, args):
                             logger.info(
                                 'Successfully downloaded %s', downloaded
                             )
+                except EnvironmentError as e:
+                    base_msg = (
+                        "Unable to install in installation directory."
+                    )
+                    no_user_msg = (
+                        "Consider using the `--user` option"
+                    )
+                    check_perms_msg = (
+                        "Check the permissions in the installation "
+                        "directory"
+                    )
+                    if e.errno == errno.EPERM:
+                        parts = [check_perms_msg]
+                        if not options.use_user_site:
+                            parts.insert(0, no_user_msg)
+                        logger.error("%s %s", base_msg, " or ".join(parts))
```

Comment:
We'll get a strange message with an upper-case letter in the middle and lacking a final period:

    Unable to install in installation directory. Consider using the `--user` option or Check the permissions
    in the installation directory